sdk/include/reactos/mc/neteventmsg.mc

```diff
@@ -213,6 +241,9 @@ ReactOS %1 %2 %3 %4.
 Language=Polish
 ReactOS %1 %2 %3 %4.
 .
+Language=Romanian
+ReactOS %1 %2 %3 %4.
```

Comment:
Just a trivial note - I do not understand why was this string duplicated in other languages.

Reply:
If I don't duplicate this string, the file will crash on compiling. Hence, just some checks will pass, all the others will fail.